site/src/atomics/spacing.md

```diff
@@ -6,7 +6,7 @@ template: standard
 
 # Spacing Atomics
 
-Spacing atomics can be used to add margin or padding on elements. They are especially helpful when composing layouts. Note that while traditional names have been kept for property values, in their declarations these classes use css logical properties. This means that while the class name may be `.margin-left-<size>`, the rule is in fact `.margin-left-<size> { margin-inline-start: <size> !important; }`. We do this to achieve great multidirectional support that requires no extra mental overhead for developers. [Read more about logical props on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties).
+Spacing atomics can be used to add margin or padding on elements. They are especially helpful when composing layouts. Note that while traditional names have been kept for property values, in their declarations these classes use css logical properties. This means that while the class name may be `.margin-left-<size>`, the rule is in fact `.margin-left-<size> { margin-inline-start: <size> !important; }`. We do this to achieve great multidirectional support that requires no extra mental overhead for developers. [Read more about logical props on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties). Note also the existence of `gap` property atomics, the documentation for which can be viewed on the [Gap Atomics page](./gap.md)
```

Comment:
Nit: add a period at the end of the sentence.

Reply:
indeed! updated